obsreport/obsreport_scraper.go

```diff
@@ -0,0 +1,117 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package obsreport
```

Comment:
Should this go in obsreport (or must it for technical reasons?) or could the scraper stuff all be kept together?

Reply:
It's more convenient adding this here as can make use of many of the helper functions for testing. I think it is also more consistent to put this here? (even thought Scrapers aren't a first class component)